lib/person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name: 'Unknow', parent_permission: true)
     @age = age
     @parent_permission = parent_permission
     @corrector = Corrector.new
+    @rentals = []
```

Comment:
You need to add `:rentals` to your `attr_accessor` collection so that `Rental` class (`lib/rental.rb` line 10) be able to access that variable, read and set it to a new value. 👍🏽

Reply:
Other than that your code is clean 💯